projects/laji/src/app/shared/footer/footer.component.ts

```diff
@@ -1,39 +1,61 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { FooterService } from '../service/footer.service';
-import { Observable } from 'rxjs';
+import { AfterViewInit, ChangeDetectionStrategy, ChangeDetectorRef, Component, Input } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
-import { Logger } from '../logger/logger.service';
-import { LajiApiService } from '../service/laji-api.service';
-import { map } from 'rxjs/operators';
-import { HeaderImage, HeaderImageService } from '../service/header-image.service';
-import { BaseDataService } from '../../graph-ql/service/base-data.service';
+import { Observable } from 'rxjs';
+import { FooterService } from '../service/footer.service';
+
+interface MaybeMultiLang { fi: string; sv?: string; en?: string };
+
+const infoLajiFiLangUrlMap: Record<string, MaybeMultiLang> = {
+  readMore: { fi: 'https://info.laji.fi/', sv: 'https://info.laji.fi/sv/', en: 'https://info.laji.fi/en/' },
```

Comment:
The other info.laji.fi links are in the translation file (with the `infoLinks.` prefix), it would be nice if all would be in the same place. But I guess it doesn't work here so well since all the links don't have translations